htdocs/societe/card.php

```diff
@@ -2936,7 +2939,43 @@ function init_supplier_categ() {
 		include DOL_DOCUMENT_ROOT.'/core/tpl/card_presend.tpl.php';
 	}
 }
-
+/**
+ * @param string $valuetoprint the value to print
+ * @param string $spanid unique id for a span
+ * @param Translate $langs the $langs variable
+ * @return string the string to print for the button
+ */
+function showValueWithCopyAndPasteButton($valuetoprint, $spanid, $langs)
```

Comment:
Can you move the function into functions.lib.php so function will be available on all pages.